Wikimate.php

```diff
@@ -495,23 +567,80 @@ public function getError()
 /**
  * Models a wiki article page that can have its text altered and retrieved.
  *
- * @author  Robert McLeod
+ * @author  Robert McLeod & Frans P. de Vries
  * @since   December 2010
  */
 class WikiPage
 {
+	/**
+	 * Use section indexes as keys in return array of {@link WikiPage::getAllSections()}
+	 *
+	 * @var integer
+	 */
 	const SECTIONLIST_BY_INDEX = 1;
+
+	/**
+	 * Use section names as keys in return array of {@link WikiPage::getAllSections()}
+	 *
+	 * @var integer
+	 */
 	const SECTIONLIST_BY_NAME = 2;
-	const SECTIONLIST_BY_NUMBER = 3;
```

Comment:
TBH, I don't know what the idea behind this third section-list key type could be. It has been in the code [since v0.4](https://github.com/hamstar/Wikimate/commit/8220b04d58b7de138724aef9e73084f7d2bece45#diff-915f1e0f55c2400eb73093b0452dbac786cbcb0a04a0f5b0db0e4adaf5b81fbbR119), but even then it wasn't used and falls into the [default clause](https://github.com/hamstar/Wikimate/commit/8220b04d58b7de138724aef9e73084f7d2bece45#diff-915f1e0f55c2400eb73093b0452dbac786cbcb0a04a0f5b0db0e4adaf5b81fbbR399) of the switch.

I propose to remove this constant altogether.

Reply:
Yeah, it seems like it would be the same as the index. I agree with removing it as it isn't used. If eventually @hamstar is able to clarify the original intent, we can reintroduce it.